deps/xmake.lua

```diff
@@ -2,3 +2,8 @@
 includes("first")
 -- Third party dependencies
 includes("third")
+add_repositories("third deps/third-repo", { rootdir = get_config("ue4ssRoot") })
```

Comment:
Should we consider a more descriptive alias for the repo? I'm fine with `third` if we add an inline comment describing what `add_repositories` is doing. 
![image](https://github.com/UE4SS-RE/RE-UE4SS/assets/7477000/628aed69-a9b1-4080-a987-160af9546cbc)

Reply:
I'll just add an inline comment, will also rename the repo to `third-party`